modules/openapi-generator/src/main/resources/C-libcurl/apiClient.c.mustache

```diff
@@ -606,7 +606,7 @@ char *sbi_base64decode (const void *b64_decode_this, int decode_this_many_bytes)
     BIO_write(mem_bio, b64_decode_this, decode_this_many_bytes); //Base64 data saved in source.
     BIO_push(b64_bio, mem_bio);          //Link the BIOs by creating a filter-source BIO chain.
     BIO_set_flags(b64_bio, BIO_FLAGS_BASE64_NO_NL);          //Don't require trailing newlines.
-    int decoded_byte_index = 0;   //Index where the next base64_decoded byte should be written.
+    decoded_byte_index = 0;   //Index where the next base64_decoded byte should be written.
```

Comment:
I think 
```c
decoded_byte_index
```
shoud change to  
```
*decoded_byte_index
```
in the function "base64decode"

Reply:
Thank @ityuhui , I changed the assigning to use local variable and then assign it to pointer as ++ is not working on pointer variables.